packages/remirror__extension-list/src/list-item-extension.ts

```diff
@@ -87,6 +87,7 @@ export class ListItemExtension extends NodeExtension<ListItemOptions> {
       }
 
       return createCustomMarkListItemNodeView({
+        view: this.store.view,
```

Comment:
There is a small possibility that the `view` here will become `undefined`. I can't reproduce it. Any idea why? @ifiokjr

Reply:
I'm not sure, let's look into it together.

Reply:
OK. Let me create a minimal reproduction in another issue.